workshop/urls.py

```diff
@@ -29,4 +32,13 @@
     path('workshops/search/', WorkshopSearchView.as_view()),
     path('workshops/search/date/', WorkshopDateSearchView.as_view()),
     path('resources/<int:pk>/', WorkshopResourceView.as_view()),
+
+    path('entities', EntityView.as_view()),
+    path('entities/<int:pk>/', EntityDetailView.as_view()),
+    path('entities/<int:pk>/tags/', EntityTagsView.as_view()),
+    path('entities/<int:pk>/toggle-subscribed/', EntitySubscriptionToggleView.as_view()),
+
+    path('tags/entities/create/', EntityTagCreateView.as_view()),
+    path('tags/entities/search/', EntityTagSearchView.as_view()),
+    path('workshops/entities/create/', EntityWorkshopCreateView.as_view()),
```

Comment:
@Vikhyath08 I think the URL patterns need to change a bit, for both clubs and entities. Maybe make another PR later to fix the URL patterns, thereby also fixing issue #42.

Reply:
Okay fair enough.
Just didn't want to modify any existing endpoints.
I'll put in a separate PR for this.